package.json

```diff
@@ -8,6 +8,16 @@
     "type": "git",
     "url": "https://github.com/juju/juju-gui.git"
   },
+  "scripts": {
+    "build": "webpack --config webpack.dev.js && cp -r node_modules/yui static && rm -rf static/yui/node_modules && ./scripts/version.sh",
+    "clean": "rm -rf static/yui static/build node_modules webapp/__pycache__",
+    "watch": "webpack --config webpack.dev.js --watch",
+    "lint-js": "eslint --rulesdir eslint-rules jujugui/static/gui/src",
```

Comment:
Would you mind adding:

```
    "lint-py": "flake8 --exclude '*env*,node_modules' && black --line-length 79 --check --exclude '(node_modules/.*|[^/]*env[0-9]?/.*)' .",
    "test-javascript": "jest",
    "test": "yarn run lint-python && yarn run lint-js && yarn run test-javascript"
```

?

Reply:
https://github.com/canonical-websites/blog.ubuntu.com/blob/master/package.json is a good example here

Reply:
Done